evaluation/examples_test.py

```
@@ -1,59 +1,99 @@
-    output_path: str = typer.Option(..., help="Path to the output CSV file"),
-    models: list[str] = typer.Option(..., help="List of models to evaluate"),
+    output_root: str = typer.Option(
+        ...,
+        "--output-root",
+        "--output-path",
+        help="Directory under which timestamped run folders (each with output.csv) are created",
+    ),
+    models: list[str] = typer.Option(
+        ...,
+        help="Model aliases to evaluate (e.g. perspective_api, openai, qwen, anthropic, minimax)",
+    ),
```

Ruff (0.15.10)

[warning] Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)